backend/api/models.py

```diff
@@ -17,34 +45,12 @@ class Item(models.Model):
         max_digits=12,
     )
     description = models.TextField()
+    history = models.ManyToManyField(History_of_Item)
```

Comment:
Everything looks good to me for the most part. My only question is about using a ManyToManyField here. It seems like it should work, but would it simplify things at all to just make it a foreign key on the History_Of_Item model since it's basically a one to many relationship?

Reply:
@MattKersey : That makes sense to me. I've implemented the chage

Steps:
- On Postman, set request type as `PUT`, endpoint to `127.0.0.1:8000/api/items/9/` (i chose item.id = 9) and add data to form-body.
- Do this several times to gather result
- Checkout output (sample below)

Note:
- I confirmed functionality on python shell (i.e. run `python3 backend/manage.py shell` on Terminal
```python
from api.models import Item
item = Item.objects.get(id=9)
item.history.all()
# <QuerySet [<History_of_Item: History_of_Item object (12)>, <History_of_Item: History_of_Item object (13)>]>
```

```json
{
    "image": "/media/api/fixtures/clothing.png",
    "name": "Clothing 1",
    "stock": 3,
    "price": "40.00",
    "description": "dnas",
    "history": [
        {
            "before_name": "Cloth 4",
            "after_name": "Clothing",
            "before_price": null,
            "after_price": null,
            "before_stock": null,
            "after_stock": null,
            "before_description": null,
            "after_description": null,
            "datetime": "2020-11-19T21:25:07.125219Z"
        },
        {
            "before_name": "Clothing",
            "after_name": "Clothing 1",
            "before_price": null,
            "after_price": null,
            "before_stock": 40,
            "after_stock": 3,
            "before_description": "Description 4",
            "after_description": "dnas",
            "datetime": "2020-11-19T21:25:20.437719Z"
        }
    ]
}
```